packages/react/src/components/Table/Table.stories.tsx

```diff
@@ -278,3 +278,71 @@ export const MultipleHeaderRows: Story = (args) => {
     </Table>
   );
 };
+
+export const WithBorder: Story = (args) => {
```

Comment:
This story has not been added to the MDX file

Reply:
Can this be a separate task as it needs some documentation? ☺️